src/OrleansRuntime/Hosting/SiloHostBuilderExtensions.cs

```diff
@@ -79,5 +82,15 @@ public static ISiloHostBuilder ConfigureLogging(this ISiloHostBuilder builder, A
         {
             return builder.ConfigureServices(collection => collection.AddLogging(loggingBuilder => configureLogging(loggingBuilder)));
         }
+
+        /// <summary>
+        /// Configure silo to use GrainBasedMembership
+        /// </summary>
+        /// <param name="builder"></param>
+        /// <returns></returns>
+        public static ISiloHostBuilder UseGrainBasedMembership(this ISiloHostBuilder builder)
```

Comment:
As we talked about, it's currently confusing, but this class is for extension methods that also exist in the generic host builder, and are not Orleans specific. There's another one for Orleans specific configuration